src/generator/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@autorest/azureresourceschema",
```

Comment:
>name [](start = 3, length = 4)

Do we need to update dependabot yaml to include this one as well? (Can also exclude any packages that we don't want it to touch.)

Reply:
Good point, updated.